draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -71,11 +170,14 @@ Security Considerations
 
 ## Additions to Existing Registries
 
-### New Entries to the COSE Header Parameters Registry
+### Tree Algorithms {#tree-alg-registry}
+
+This document requests IANA to add the following new value to the 'Tree Algorithms' registry:
+
 
-This document requests IANA to add new values to the 'COSE
-Algorithms' and to the 'COSE Header Algorithm Parameters' registries
-in the 'Standards Action With Expert Review category.
+* Identifier: TBD_1
+* Tree Algorithm: ccf_ledger
+* Reference: This document
```

Comment:
This is indeed all you would need to register.. unless there are more "tree algorithm" specific proof types for CCF

Reply:
Seems ok to call the algorithm a "ledger" here.

Reply:
lgtm at this stage

Reply:
@OR13 I would leave it as here but move the discussion to the latest COMETRE PR to see if we prefer to register multiple "tree algs" for different proof types or if we prefer to make "proof type" its own registry